livestream/index.html

```diff
@@ -0,0 +1,313 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>Tickets - ETHIndia | August 10 - 12, 2018</title>
```

Comment:
Would recommend a few changes before we merge this to master.

[ ] Changing the title here to `Livestream - ETHIndia | August 10 - 12, 2018`